src/store/ducks/opensrp/files/index.ts

```diff
@@ -0,0 +1,122 @@
+import { keyBy, values } from 'lodash';
```

Comment:
It seems that this reducer is not a "files" reducer.  It seems to be specifically for client related files.  Can we make that clearer in its naming?